ducktape/command_line/main.py

```diff
@@ -60,15 +60,23 @@ def main():
     # in which all test results will be stored
     session_id = generate_session_id(ConsoleConfig.SESSION_ID_FILE)
     results_dir = generate_results_dir(session_id)
-    test_session_context = TestSessionContext(session_id, results_dir)
+    cluster = VagrantCluster()
```

Comment:
Not related to this patch, but should we file an issue to add command line options to choose the cluster type? Not immediately necessary for us, but we'll need it before we can suggest using this in Kafka. This type of small task seems like the kind of thing we should record issues for as we encounter them to make sure they aren't forgotten.